drape_frontend/visual_params.hpp

```diff
@@ -19,12 +19,16 @@ double constexpr kMaxVisualScale = 4.0;
 class VisualParams
 {
 public:
-  static double const kMdpiScale;
-  static double const kHdpiScale;
-  static double const kXhdpiScale;
-  static double const k6plusScale;
-  static double const kXxhdpiScale;
-  static double const kXxxhdpiScale;
+  static double constexpr kMdpiScale = 1.0;
```

Comment:
There is no need to use `static` with constexpr.

Reply:
GCC complains
```
/home/pastk/dev/OM/src/drape_frontend/visual_params.hpp:22:33: error: non-static data member ‘kMdpiScale’ declared ‘constexpr’
   double constexpr kMdpiScale = 1.0;
```

Reply:
My bad, in class members it's required!